README.md

```diff
@@ -398,20 +398,20 @@ Never leave commented-out code in our codebase.
 
 ### Method definitions
 
-* Use `def` with parentheses when there are arguments. Omit the
-  parentheses when the method doesn't accept any arguments.
+* Use `def` with parentheses when there are parameters. Omit the
+  parentheses when the method doesn't accept any parameters.
 
      ```Ruby
      def some_method
        # body omitted
      end
 
-     def some_method_with_arguments(arg1, arg2)
+     def some_method_with_parameters(arg1, arg2)
        # body omitted
      end
      ```
 
-* Do not use default arguments. Use an options hash instead.
+* Do not use default parameters. Use an options hash instead.
```

Comment:
Ahh yes, the hash _inside_ the method is what you're referring to, **Not** `options = {}`?